subiquity/server/controllers/install.py

```diff
@@ -147,14 +155,11 @@ async def install(self, *, context):
 
             self.app.update_state(ApplicationState.RUNNING)
 
-            config_location = self.write_config()
-
             if os.path.exists(self.model.target):
                 await self.unmount_target(
                     context=context, target=self.model.target)
 
-            await self.curtin_install(
-                context=context, config_location=config_location)
+            await self.curtin_install(context=context)
```

Comment:
looks like this needs a config_location arg

Reply:
Of course when I test, then make a "trivial" refactoring, I break stuff. Fix pushed (not tested yet though)